bin/codebasin

```diff
@@ -272,23 +247,23 @@ def main():
                 excludes = analysis_toml["codebase"]["exclude"]
                 codebase["exclude_patterns"] += excludes
 
+        for name in args.platforms:
+            if name not in analysis_toml["platform"].keys():
+                raise RuntimeError(
```

Comment:
To be pedantic, I think this would throw a `KeyError`

Reply:
Pedantic is good.  Fixed in ff83d44f7895d4e19a1df4d131162e8fea5ce9d8.